python/plugins/db_manager/db_plugins/info_model.py

```diff
@@ -91,12 +98,18 @@ def toHtml(self):
         else:
             ret.append(HtmlSection(QApplication.translate("DBManagerPlugin", 'Connection details'), conn_details))
 
-        # database information
+        # general information
         general_info = self.generalInfo()
         if general_info is None:
             pass
         else:
             ret.append(HtmlSection(QApplication.translate("DBManagerPlugin", 'General info'), general_info))
+        # database information
+        database_info = self.databaseInfo()
+        if database_info is None:
+            pass
+        else:
+            ret.append(HtmlSection(QApplication.translate("DBManagerPlugin", 'Database informations'), database_info))
```

Comment:
```suggestion
        if database_info is not None:
            ret.append(HtmlSection(QApplication.translate("DBManagerPlugin", 'Database information'), database_info))
```

"information" is always singular. Also simplifying the if condition a little bit.
We could even skip the "is not not None", up to you.